src/hyperloglog.c

```diff
@@ -1334,12 +1334,10 @@ void hllMergeDense(uint8_t *reg_raw, const uint8_t *reg_dense) {
         }
     }
 #endif
-#ifdef __aarch64__
-    if (HLL_REGISTERS == 16384 && HLL_BITS == 6) {
-        if (HLL_USE_NEON) {
-            hllMergeDenseNEON(reg_raw, reg_dense);
-            return;
-        }
+#if __aarch64__ && HAVE_ARM_NEON && HLL_REGISTERS == 16384 && HLL_BITS == 6
```

Comment:
Build warning/error:

```
hyperloglog.c:1337:5: error: "__aarch64__" is not defined, evaluates to 0 [-Werror=undef]
 1337 | #if __aarch64__ && HAVE_ARM_NEON && HLL_REGISTERS == 16384 && HLL_BITS == 6
      |     ^~~~~~~~~~~
```

We only need to check `HAVE_ARM_NEON`?

```suggestion
#if HAVE_ARM_NEON && HLL_REGISTERS == 16384 && HLL_BITS == 6
```